examples/persistent-volumes/volumes/local-01.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolume
+apiVersion: v1beta3
+metadata:
+  name: pv0001
+  labels:
+    type: local
```

Comment:
Having a local, temporary directory referred to as a "Persistent" volume will be a source of confusion to users.  I have no suggested improvement yet - let me finish reading first :-)